src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/ConnectionPoolTest.cs

```diff
@@ -280,12 +283,5 @@ private static InternalConnectionWrapper ReplacementConnectionUsesSemaphoreTask(
 
             return internalConnection;
         }
-
```

Comment:
Please bring this back which is needed for the `ReclaimEmancipatedOnOpenTest`.

Reply:
Done